deepmd/pt/model/task/property.py

```diff
@@ -126,6 +138,7 @@ def output_def(self) -> FittingOutputDef:
                     r_differentiable=False,
                     c_differentiable=False,
                     intensive=self.intensive,
+                    sub_var_name=self.property_name,
```

Comment:
why not setting `self.var_name` as the property_name?  Then we do not need to maintain `sub_var_name` feature for the `OutputVariableDef`.

Reply:
That is because `property_name` may be a list when we want to fit multiple properties which store in different `.npy` files.